matlab/schemas/+stimulus/+conf/Varma.m

```diff
@@ -2,38 +2,123 @@
 control = stimulus.getControl;
 control.clearAll()   % clear trial queue and cached conditions.
 
+paramsM.fps             = 60;
+paramsM.duration        = 180;  % to be changed to 1800s
```

Comment:
I recommend against making very long trials.  In our current framework, we do not log partially completed trials.  So if for some reason we need to interrupt the experiment, the ongoing trial is lost.  Therefore, the longest trials we have are ~60 seconds in long experiments.

Of course, we could change the framework to allow processing partially completed trials, increasing the amount of programming and testing required to handle special cases such as interrupted trials.